test/powershell/Modules/Microsoft.PowerShell.Management/Get-Item.Tests.ps1

```diff
@@ -28,10 +28,10 @@ Describe "Get-Item" -Tags "CI" {
     }
 
     It "Using -literalpath should find no additional files" {
-        $null = New-Item -type file "$TESTDRIVE/file[abc].txt"
-        $null = New-Item -type file "$TESTDRIVE/filea.txt"
+        $null = New-Item -type file "$TESTDRIVE\file[abc].txt"
```

Comment:
I wonder how is this related to the PR? Please undo all unrelated changes.

Reply:
@iSazonov if you look at the commit message where this was made it is to clean up a discrepancy in the tests that was uncovered when a prior test failed. So no I won't undo this commit

Reply:
You would have saved reviewers time if you had added a link to the post you are talking about.

The error occurs only in the new test you added. This fact indicates that only the following lines should be corrected because these tests are only Windows test:

```powershell
            $altStreamPath = "$TESTDRIVE\altStream.txt"
            $altStreamDirectory = "$TESTDRIVE\altstreamdir"
            $noAltStreamDirectory = "$TESTDRIVE\noaltstreamdir"
```

Changes in other tests should be reverted because they are not related to this PR.

Reply:
@iSazonov I can waste time in issuing an additional PR correcting this error across other test files or I can save time by including it within this PR seeing as it it related to the PR. If you don't agree with me that's fine, but this change in my opinion makes sense.

Reply:
Have you analyzed all the code paths? What if this is done intentionally? You are forcing every reviewer to spend time analyzing the history of these tests even though they have nothing to do with this PR.

If you really think that these tests contain a bug, then make a new PR to improve them.

By making irrelevant changes you waste both your time and the time of code reviewers.

Reply:
@kilasuit I agree with @iSazonov here.  I appreciate you trying to fix issues to the tests, but if they are unrelated to the PR, it makes it confusing why they are being changed and implies the PR caused some problem that requires this change.  PowerShell is expected to normalize the slashes, so I don't understand why this change is necessary at all unless there's a problem in PowerShell handling the slashes.

My suggestion is to revert the unrelated test changes and create a new PR referencing the issue being addressed to provide more context to the reviewers why the change needs to be made.  Thanks!

Reply:
@SteveL-MSFT based on your comments, I've undone the commit that changed the rest of the tests and force pushed my branch with only the changes that I found were required for the tests to pass in this PR to be able to be merged.